pyproject.toml

```diff
@@ -1,6 +1,109 @@
 [build-system]
-requires = ["setuptools>=66", "wheel"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "geti-sdk"
+description = "Software Development Kit for the Intel® Geti™ platform"
+authors = [
+    {name = "Intel Corporation"}
+]
+maintainers = [
+    {name = "Leonardo Lai", email = "leonardo.lai@intel.com"},
+    {name = "Alexander Barabanov", email = "alexander.barabanov@intel.com"},
+]
+readme = "README.md"
+license-files = ["LICENSE", "CODE_OF_CONDUCT.md"]
+keywords = ["intel", "geti", "computer vision", "deep learning", "machine learning"]
+
+requires-python = ">=3.9,<3.13"
+dynamic = ["version"]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Operating System :: OS Independent",
+    "License :: OSI Approved :: Apache Software License",
+    "Development Status :: 5 - Production/Stable",
+]
+
+dependencies = [
+    "certifi~=2025.6.15",
+    "cryptography~=45.0",
+    "datumaro~=1.10",
+    "defusedxml~=0.7",
+    "imageio-ffmpeg~=0.6",
+    "ipython~=8.18",
+    "joblib~=1.5",
+    "numpy~=1.26.4",
+    "omegaconf~=2.3",
+    "opencv-python~=4.11",
+    "openvino-model-api~=0.2.5.1",
+    "openvino~=2024.6",
+    "orjson~=3.10",
+    "ovmsclient~=2023.1",
+    "pathvalidate~=2.5",
+    "pillow~=11.2",
+    "protobuf~=3.20",
+    "python-dotenv~=1.1",
+    "requests~=2.32",
+    "semver~=3.0",
+    "simplejson~=3.20",
+    "tqdm~=4.67",
+    "urllib3~=1.26",
+]
+
+[dependency-groups]
+dev = [
+    "pre-commit~=4.2",
+    "pytest~=8.3",
+    "pytest-cov~=6.0",
+    "pytest-env~=1.1",
+    "pytest-html~=4.1",
+    "pytest-mock~=3.10",
+    "pytest-recording~=0.13",  # pinned due to https://github.com/kiwicom/pytest-recording/issues/174
+    "ruff~=0.12",
+    "snowballstemmer~=2.2",  # pinned because pytest-recording subdeps are loose
+    "vcrpy~=7.0",
+]
+
+[project.optional-dependencies]
+docs = [
+    "myst-parser~=2.0",
+    "sphinx~=7.2",
+    "sphinx-rtd-theme~=1.3",
+]
+notebooks = [
+    "ipywidgets~=8.1",
+    "jupyterlab~=4.0",
+    "jupyter-core~=4.11",
+    "mistune~=2.0",
+    "pandas~=2.3",
+]
+
+[project.urls]
+Documentation = "https://docs.geti.intel.com/"
+Repository = "https://github.com/open-edge-platform/geti-sdk"
+Issues = "https://github.com/open-edge-platform/geti-sdk/issues"
+Changelog = "https://github.com/open-edge-platform/geti-sdk/releases"
+
+[tool.hatch.version]
+path = "geti_sdk/_version.py"
+
+[tool.hatch.build.targets.sdist]
+only-include = ["geti_sdk", "README.md", "LICENSE", "CODE_OF_CONDUCT.md", "pyproject.toml"]
+
+[tool.hatch.build.targets.wheel]
+packages = ["geti_sdk"]
+
+[tool.uv]
+override-dependencies = [
+    # Exclude unnecessary subdependencies from datumaro
```

Comment:
Note: datumaro components will be deprecated and removed soon [ITEP-70633](https://jira.devtools.intel.com/browse/ITEP-70633)